repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -1702,6 +1709,52 @@ export const upsertFareDayEnd = async (nocCode: string, fareDayEnd: string): Pro
         throw new Error(`Could not insert fare day end into the fareDayEnd table. ${error}`);
     }
 };
+export const getCapExpiry = async (nocCode: string): Promise<string | undefined> => {
+    logger.info('', {
+        context: 'data.auroradb',
+        message: 'retrieving cap expiry for given nocCode',
+        nocCode,
+    });
+
+    try {
+        const queryInput = `
+            SELECT contents
+            FROM caps
+            WHERE noc = ?
+            AND isExpiry = 1
+        `;
+
+        const queryResults = await executeQuery<{ contents: string }[]>(queryInput, [nocCode]);
```

Comment:
Will this function just throw an error if the user has no capExpiry?

Reply:
it does not throw any error in the console.